internal/api/api.go

```diff
@@ -216,6 +219,27 @@ func (api *idler) Info(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 	}
 }
 
+func (api *idler) Reset(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	logger := log.WithFields(log.Fields{"component": "api", "function": "Reset"})
+
+	openShiftAPI, openShiftBearerToken, err := api.getURLAndToken(r)
+	if err != nil {
+		logger.Error(err)
+		writeResponse(w, http.StatusBadRequest, fmt.Sprintf("{\"error\": \"%q\"}", err))
+		return
+	}
+
+	err = api.openShiftClient.Reset(openShiftAPI, openShiftBearerToken, ps.ByName("namespace"))
```

Comment:
Do you know if 
```go
	ns := strings.TrimSpace(ps.ByName("namespace"))
	if ns == "" {
		err = errors.New("Missing mandatory param namespace")
		respondWithError(w, http.StatusBadRequest, err)
		return
	}
```
is needed?

Reply:
Without namespace we can't know which jenkins instance to apply this api